src/ngraph/frontend/onnx_import/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ add_library(onnx_import_interface OBJECT
         onnx_utils.hpp
         onnx_utils.cpp)
 
-add_library(onnx_import STATIC
+add_library(onnx_import_core STATIC
```

Comment:
Why do you create a static library `onnx_import_core`? Looks like you can create only `onnx_importer` shared library.

The `onnx_import_core` is linking only with the `onnx_importer` shared library and doesn't use anymore.

Reply:
We have a separation between interface and implementation in onnx importer.
Since onnx_import_core is a `PRIVATE` library for onnx_importer its symbols don't pollute the main library interface.

Reply:
But in general in the line 266 I see that we buid the library with visibility hidden flag, in this case the library interface contains only expoeted methods.

Reply:
I mean that visibility hidden allows to hide all internal symbols and you don't need to create `onnx_import_core` library. With visibility hidden only exported symbols will be exists in the interface of `onnx_importer` library.

Reply:
Done